pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/mapper/HadoopSegmentCreationMapReduceJob.java

```diff
@@ -191,26 +199,48 @@ protected void map(LongWritable key, Text value, Context context) throws IOExcep
 
       try {
         String segmentName =
-            createSegment(_inputFilePath, schema, Integer.parseInt(lineSplits[2]), hdfsInputFilePath, localInputDataDir,
-                fs);
+                createSegment(_inputFilePath, schema, Integer.parseInt(lineSplits[2]), hdfsInputFilePath, localInputDataDir,
+                        fs);
         LOGGER.info(segmentName);
         LOGGER.info("Finished segment creation job successfully");
       } catch (Exception e) {
         LOGGER.error("Got exceptions during creating segments!", e);
       }
 
       context.write(new LongWritable(Long.parseLong(lineSplits[2])), new Text(
-          FileSystem.get(_properties).listStatus(new Path(_localHdfsSegmentTarPath + "/"))[0].getPath().getName()));
+              FileSystem.get(_properties).listStatus(new Path(_localHdfsSegmentTarPath + "/"))[0].getPath().getName()));
       LOGGER.info("Finished the job successfully");
     }
 
     protected void setSegmentNameGenerator(SegmentGeneratorConfig segmentGeneratorConfig, Integer seqId,
-        Path hdfsAvroPath, File dataPath) {
+                                           Path hdfsAvroPath, File dataPath) {
```

Comment:
could we also add a check to ensure that the format is either epoch or simple date and throw an exception if it isn't?

Reply:
default is epoch and if simple date format is not right, it would anyways throw exception as its in try

Reply:
For codestyle reasons so that people can understand what the allowed formats are, I would prefer if you checked for this up front, with a comment that only epoch and simpledate time formats are allowed